tests/ZendTest/Stdlib/OptionsTest.php

```diff
@@ -112,4 +113,36 @@ public function testExceptionMessageContainsActualUsedSetter()
             'foo bar' => 'baz',
         ));
     }
+
+	public function testIssetFailsWhenNoGetter()
+	{
+		$options = new TestOptionsWithoutGetter(array('foo' => 'bar'));
+		$this->assertFalse(isset($options->foo));
+
+	}
+
+	public function testIssetDoesNotThrowExceptionWhenNoGetter()
+	{
+		$options = new TestOptionsWithoutGetter();
+
+		$ex = null;
+
+		try {
+			isset($options->foo);
+		} catch (\BadMethodCallException $ex) {}
+
+		$this->assertNull($ex, 'Unexpected BadMethodCallException');
```

Comment:
Use **phpunit** assert for exception `$this->setExpectedException('BadMethodCallException')`